### DISCUSSION TITLE
Support for ArangoDB

### DISCUSSION THREAD

**Discussion by iyobo** [mikro-orm/mikro-orm#2120] [Ideas]
[ArangoDB](https://www.arangodb.com/) is a solid multi-paradigm database (graph, document, key:value).
It would be great if MikroORM supported it.

**Comment by B4nan** [comment]
If you wanna see that happen, try to contribute. Personally I have no plans to work on any additional drivers in near future, and historically there was no driver contributed by the community, just some unfinished efforts, so better not to expect someone else will do it 😉
